data/strategies/StrategyMakerETHDAIDelegate ETH-A.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyMakerETHDAIDelegate ETH-A",
+  "description": "Locks ETH at MakerDAO via ETH-A to mint DAI. This newly minted DAI is then deposited into the v2 DAI yVault.",
```

Comment:
Do we want to say something about the debt ratio? or may tiny bit longer explanation of this strat?